README.md

```diff
@@ -168,7 +159,8 @@ Site:13DNDk..bhC2 Successfuly published to 3 peers
 
 ## Help keep this project alive
 
-- Bitcoin: 1QDhxQ6PraUZa21ET5fYUCPgdrwBomnFgX
+- Bitcoin: [1QDhxQ6PraUZa21ET5fYUCPgdrwBomnFgX](bitcoin:1QDhxQ6PraUZa21ET5fYUCPgdrwBomnFgX)
+<img width="250px" src="http://i.imgur.com/FVSnQOn.png"/>
```

Comment:
It's probably okay to upload the image of the QR code to the repo and display it instead of relying on imgur